web/themes/contrib/civictheme/civictheme.post_update.php

```diff
@@ -619,3 +620,97 @@ function civictheme_post_update_enable_focal_point_configurations_2(): void {
+    $blocks = \Drupal::entityTypeManager()
+      ->getStorage('block')
+      ->loadByProperties(['theme' => $theme, 'region' => 'sidebar']);
+    if ($blocks) {
+      foreach ($blocks as $block) {
+        $block->setRegion('sidebar_top_left');
+        $block->set('status', TRUE);
+        $block->save();
+      }
+    }
+    // Check if <theme_name>_side_navigation block exists.
+    $sideNavigationBlock = \Drupal::entityTypeManager()
+      ->getStorage('block')
+      ->loadByProperties(['theme' => $theme, 'id' => $theme . '_side_navigation']);
+    if ($sideNavigationBlock) {
+      foreach ($sideNavigationBlock as $block) {
+        // Update the region to 'sidebar_top_left'.
+        $block->setRegion('sidebar_top_left');
+        $block->set('status', TRUE);
+        $block->save();
+      }
+    }
+  }
+
+  \Drupal::messenger()->addMessage('All blocks in the "sidebar" region have been moved to "sidebar_top_left".');
+}
+
+/**
+ * Enables "3 column" layout for the Page content type.
+ * 
+ * @SuppressWarnings(PHPMD.ElseExpression)
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+function civictheme_post_update_enable_three_column_layout(): void {
+  // Load the current display configuration for the Page content type.
+  $entity_display = \Drupal::entityTypeManager()
+    ->getStorage('entity_view_display')
+    ->load('node.civictheme_page.default');
+
+  // Check if the display is already set to use the "3 column" layout.
+  if ($entity_display) {
+    $entity_view_mode_restriction = $entity_display->getThirdPartySetting('layout_builder_restrictions', 'entity_view_mode_restriction', []);
+    if (!empty($entity_view_mode_restriction['allowed_layouts']) && !in_array('civictheme_three_columns', $entity_view_mode_restriction['allowed_layouts'])){
+      
+      $update_config = TRUE;
+      if (!empty($entity_view_mode_restriction['allowlisted_blocks'])) {
+        $update_config = FALSE;
+      }
+
+      if ($update_config){
+
+        // Update the configuration to use the "3 column" layout.
+        $entity_view_mode_restriction['allowed_layouts'] = [];
+        $entity_view_mode_restriction['allowed_layouts'][] = 'civictheme_three_columns';
+        $entity_display->setThirdPartySetting('layout_builder_restrictions', 'entity_view_mode_restriction', $entity_view_mode_restriction);
+        $entity_display->save();
+        \Drupal::messenger()->addMessage('Enabled "3 column" layout for the Page content type.');
+
+        $theme_config_path = \Drupal::service('extension.list.theme')->getPath('civictheme');
+        $theme_config_file = $theme_config_path . '/config/install/core.entity_view_display.node.civictheme_page.default.yml';
+        
+        if (file_exists($theme_config_file)) {
+          $theme_config = \Drupal::service('config.factory')->getEditable('core.entity_view_display.node.civictheme_page.default');
+          $yaml_data = Yaml::parseFile($theme_config_file);
+          $theme_config->setData($yaml_data);
```

Comment:
this will override **all** display settings of the page, but we need to override only the layout settings.

Please update to only override layout settings.

Maybe the config for the layout settings can still come from the config file loaded into `$theme_config`, but use only a subset of the configs.